scripts/build-rancherd

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+cd $(dirname $0)
+K8S_VERSION=$(grep 'ARG RKE2_VERSION=' ../package/Dockerfile.runtime | sed 's/.*RKE2_VERSION=//')
+curl -sfL https://raw.githubusercontent.com/rancher/rke2/${K8S_VERSION/-/+}/scripts/version.sh > rke2-version
+source rke2-version
```

Comment:
😬 
This script also introspects the current git repo, hopefully that doesn't have any unwanted side effects.

Reply:
Removed.